docs/source/config_files.rst

```diff
@@ -30,6 +55,14 @@ You can also install files from pip in your ``environment.yml`` as well.
 Our example `environment.yml <https://github.com/binder-examples/python-conda_pip/blob/master/environment.yml>`_
 shows how one can specify a conda environment for repo2docker.
 
+You can also specify which Python version you are using with
+``environment.yml``. By default, ``repo2docker`` will assume you are using
+Python 3.6 with your ``environment.yml`` unless you include the version of
+Python in the file.  ``conda`` supports Python versions 2.7, 3.4, 3.5, and 3.6.
+``repo2docker`` support is best with Python 2.7, 3.5, and 3.6. If you include
```

Comment:
A Python version specified in an environment.yml takes precedence over all other configuration files. If a Python version is included in environment.yml and also exists in another file, such as runtime.txt, **repo2docker will ignore Python version in the runtime.txt file.

Reply:
I think removing emphasis from `runtime.txt` is a good idea. If we can funnel people towards using `environment.yml` I think that would be a good thing. The conda buildpack is somehow the most powerful of all of our Pokemon.